integration-tests/docker-base/jdk11/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Based on the SequenceIQ hadoop-docker project hosted at
+# https://github.com/sequenceiq/hadoop-docker, and modified at
+# the Apache Software Foundation (ASF).
```

Comment:
this looks like it now includes some stuff from an [AdoptOpenJdk](https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/11/jdk/ubuntu/Dockerfile.hotspot.releases.slim) or similar dockerfile, could you update this comment section to include where this came from?

Reply:
Done